src/components/settings/PushNotifications/PushNotificationsBanner/index.tsx

```diff
@@ -86,69 +87,146 @@ export const _getSafesToRegister = (
   return { [chainId]: newlyAddedSafes }
 }
 
-const TrackBanner = (): null => {
-  const hasTracked = useRef(false)
+type BannerProps = {
+  children: ReactElement
+  onDismiss?: () => void
+  onCustomize?: () => void
+  onEnableAll?: () => void
+}
 
-  useEffect(() => {
-    if (hasTracked.current) {
-      return
-    }
+const Banner = ({ children, onDismiss, onEnableAll, onCustomize }: BannerProps) => {
+  const { chainName = '' } = useCurrentChain() || {}
+  const { query } = useRouter()
 
-    trackEvent(PUSH_NOTIFICATION_EVENTS.SHOW_BANNER)
-    hasTracked.current = true
-  }, [])
+  return (
+    <CustomTooltip
+      className={css.banner}
+      title={
+        <Grid container className={css.container}>
+          <Grid item xs={3}>
+            <Chip label="New" className={css.chip} />
+            <SvgIcon component={PushNotificationIcon} inheritViewBox fontSize="inherit" className={css.icon} />
+          </Grid>
 
-  return null
+          <Grid item xs={9}>
+            <Typography variant="subtitle2" fontWeight={700}>
+              Enable push notifications
+            </Typography>
+
+            <IconButton onClick={onDismiss} className={css.close}>
+              <SvgIcon component={CloseIcon} inheritViewBox color="border" fontSize="small" />
+            </IconButton>
+
+            <Typography mt={0.5} mb={1.5} variant="body2">
+              Get notified about pending signatures, incoming and outgoing transactions for all Safe Accounts on{' '}
+              {chainName} when {`Safe{Wallet}`} is in the background or closed.
+            </Typography>
+
+            {/* Cannot wrap singular button as it causes style inconsistencies */}
+            <CheckWallet>
+              {(isOk) => (
+                <div className={css.buttons}>
+                  <Button
+                    variant="contained"
+                    size="small"
+                    className={css.button}
+                    onClick={onEnableAll}
+                    disabled={!isOk}
+                  >
+                    Enable all
+                  </Button>
+
+                  <Link passHref href={{ pathname: AppRoutes.settings.notifications, query }} onClick={onCustomize}>
+                    <Button variant="outlined" size="small" className={css.button}>
+                      Customize
+                    </Button>
+                  </Link>
+                </div>
+              )}
+            </CheckWallet>
```

Comment:
As we are no longer showing the banner when no wallet is connected, we don't need to wrap this in `CheckWallet`.